### DISCUSSION TITLE
Display a toast after submitting a form with errors

### DISCUSSION THREAD

**Discussion by wilav-dev** [react-hook-form/react-hook-form#8481] [General]
Is there any onError to display a toast "Please review invalid fields". Because want to show it once and not on every field update.

**Comment by wilav-dev** [comment]
Sorry, I found it since v6 in handleSubmit. I was using v5 when created my form wrapper

**Comment by pedro757** [comment]
FYI you could use onInvalid to display a toast
```
form.handleSubmit(onValid: () => void, onInvalid: () => void): void
```
